### PR TITLE
Requirements are updated and Capitalized examples task name

### PR DESCRIPTION
**Pull request by Rizwan-Hasan**
- Requirements are updated  and tested using
    - Python 3.5.7
    - Python 3.6.5
    - Python 3.7.4
    - Python 3.8.7
- Capitalized the examples task name

### CODE REVIEW

File: examples/frameworks/pytorch/requirements.txt [hunk 1]
```diff
@@ -1,7 +1,8 @@
...
 tensorboard>=1.14.0
-torch>=1.1.0
-torchvision>=0.3.0
+torch >= 1.1.0 ; python_version >= '3.8'
+torch <= 1.5.1 ; python_version < '3.8'
+torchvision
 tqdm
-clearml
\ No newline at end of file
+jsonschema==3.2.0 ; python_version <= '3.5'
```
Comment (shomratalon) [inline examples/frameworks/pytorch/requirements.txt:8]: what about the case of python_version > '3.5'?

**Review by shomratalon — CHANGES_REQUESTED** [review]

File: examples/frameworks/pytorch/requirements.txt [hunk 2]
```diff
@@ -1,7 +1,8 @@
 matplotlib
 tensorboardX
 tensorboard>=1.14.0
-torch>=1.1.0
-torchvision>=0.3.0
+torch >= 1.1.0 ; python_version >= '3.8'
+torch <= 1.5.1 ; python_version < '3.8'
+torchvision
 tqdm
-clearml
```
[the remark this answers is not in the document]
Comment (Rizwan-Hasan) [reply examples/frameworks/pytorch/requirements.txt:7]: Mistakenly hadn't selected the clearml while copying from the testing folder.

File: examples/frameworks/pytorch/requirements.txt (hunk 1 above)
Comment (Rizwan-Hasan) [reply examples/frameworks/pytorch/requirements.txt:8]: In my test, I saw pip ignored the condition and instead install suitable jsonschema automatically which is higher than the version 3.2.0.

**Review by Rizwan-Hasan — COMMENTED** [review]
added clearml

**Comment by Rizwan-Hasan** [comment]
ol

**Comment by Rizwan-Hasan** [comment]
- Capitalized the examples task name

**Review by JDennisJ — APPROVED** [review]

**Pull request merged**